src/originalBudgetTrackingApp/commonTypes.ts

```diff
@@ -12,8 +13,17 @@ export enum OutputVendorName {
   GOOGLE_SHEETS = 'googleSheets'
 }
 
+export type ExportTransactionsParams = {
+  transactionsToCreate: EnrichedTransaction[];
+  startDate: Date;
+  outputVendorsConfig: Config['outputVendors'];
+  eventEmitter: BudgetTrackingEventEmitter
+}
```

Comment:
```suggestion
export type ExportTransactionsParams = {
  transactionsToCreate: EnrichedTransaction[];
  startDate: Date;
  outputVendorsConfig: Config['outputVendors'];
}
```
I think the `eventEmitter` should be a different argument because it not related to the exporter itself. It is not part of the exporter input.